### ISSUE TITLE
Easy search attributes in dev server

### ISSUE THREAD

**Issue by cretz** [temporalio/features#494]
### Describe the solution you'd like

* CLI dev server easy search attribute registration via CLI
* Languages using CLI dev server expose search attribute option
* Languages using time-skipping Java test server expose search attribute options that become operator service calls to test server

### Per-SDK Tickets

- [ ] Temporal CLI - https://github.com/temporalio/cli/issues/21
- [ ] Go - TODO after CLI done/tagged
- [x] Java - Already done
- [x] Core - N/A
- [ ] TypeScript - TODO after CLI done/tagged
- [ ] Python - TODO after CLI done/tagged
- [ ] .NET - TODO after CLI done/tagged
- [ ] PHP - TODO after CLI done/tagged

**Comment by cretz** [comment]
Adding this to SDKs needs to wait on https://github.com/temporalio/temporal/issues/6475 being available and used by the CLI.
